src/rust/engine/src/nodes.rs

```diff
@@ -1065,7 +1065,8 @@ impl NodeKey {
   fn workunit_level(&self) -> Level {
     match self {
       NodeKey::Task(ref task) => task.task.display_info.level,
-      _ => Level::Debug,
+      NodeKey::DownloadedFile(..) => Level::Debug,
```

Comment:
I set this to `Debug` because downloading does seem like something more note-worthy than reading from the local FS.

Reply:
It's possible that we want other Nodes to be Debug too.

https://github.com/pantsbuild/pants/blob/f80825316ee5bae2b962af66ac7471d68b23c08f/src/rust/engine/src/nodes.rs#L1022-L1031

Maybe `ReadLink`? Note that `MultiPlatformExecuteProcess` will have a separate log level for the underlying `Process`(es) belonging to the MPEP, which defaults to Info and can be overridden in Python in the `Process` constructor.

Reply:
Agreed re downloads should be debug, filesystem operations (including readlink) should be trace, just due to sheer volume.